workers/tasks/export/pandoc.js

```diff
@@ -1,35 +1,78 @@
+import path from 'path';
 import fs from 'fs';
 import nodePandoc from 'node-pandoc';
 import YAML from 'yaml';
 
+import { getLicenseBySlug } from 'shared/license';
 import { getTmpFileForExtension } from './util';
 
-const dataRoot = process.env.NODE_ENV === 'production' ? '/app/.apt/usr/share/pandoc/data ' : '';
+const dataRoot = path.dirname('templates');
```

Comment:
My guess is that this is intended to resolve the absolute path of the `templates` directory, but this function just shaves off the part of the pathstring inferred to be the directory path and will always return `"."`, no matter what — that happens to work in this case, but I would rather just write `"."` into the `--template` flag below, or use `__dirname`.

Reply:
👍